user/deployment/releases.md

```diff
@@ -98,7 +98,8 @@ The GitHub-generated tags are of the form `untagged-*`, where `*` is a random
 hex string.
 Notice that this tag is immediately available on GitHub, and thus
 will trigger a new Travis CI build, unless it is prevented by
-[other means](/user/customizing-the-build/#skipping-a-build).
+other means; for instance, by 
+[blocklisting `/^untagged/`](/user/customizing-the-build/#skipping-a-build).
```

Comment:
I think the link should have been https://docs.travis-ci.com/user/customizing-the-build/#safelisting-or-blocklisting-branches

Reply:
Thanks. I fixed this.